IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_ProtectiveDeviceTypeAntiArcingDevice/DocPropertySet.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_ProtectiveDeviceTypeAntiArcingDevice" UniqueId="c8afd89c-b11c-24ac-9053-b0ee60750e1a" ApplicableType="IfcProtectiveDevice/ANTI_ARCING_DEVICE" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

Comment:
Just noticed this seems to be in the wrong PR?? this change needs to be removed as it alters changes already applied in #161